CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ### Dependencies
 - Update nebula-publishing-plugin to 19.2.0 ([#5704](https://github.com/opensearch-project/OpenSearch/pull/5704))
 ### Changed
+- Use ReplicationFailedException instead of OpensearchException in ReplicationTarget ([#4725](https://github.com/opensearch-project/OpenSearch/pull/4725))
```

Comment:
Note that the [changelog entry on main](https://github.com/opensearch-project/OpenSearch/blob/f7dc32e5e12c2ed3fd60abbcac793d11706af7e2/CHANGELOG.md?plain=1#L49) will be in the wrong spot if this is backported to 2.x. Can you update the changelog on main after this is merged into 2.x?

Reply:
Sure @andrross, thanks for pointing this out.

Reply:
@andrross : Raised PR for this https://github.com/opensearch-project/OpenSearch/pull/5960. Please have a look.